testing/it/http/src/test/java/com/adobe/cq/wcm/core/components/it/http/PageIT.java

```diff
@@ -55,4 +64,47 @@ public void testBrandSlug() throws ClientException {
         content = adminAuthor.doGet("/content/core-components/simple-page/simple-subpage.html", 200).getContent();
         GraniteAssert.assertRegExFind(content, "<title>Simple SubPage \\| Core Components</title>");
     }
+
+    @Test
+    public void testPWAProperties() throws ClientException {
```

Comment:
These tests should probably be skipped on 6.4 and 6.5 if the PWA features were not backported. @msagolj, any pointers here?

Reply:
- create an Empty Interface named e.g. `IgnoreOn64` and `IgnoreOn65`
- Mark tests to ignore with `@Category({IgnoreOn64.class, IgnoreOn65})`
- In the Jenkinsfile we then have to add a` .withCategoryFilter("IgnoreOn64")` etc. for the corresponding `IntergrationTestRun `configurations.

Reply:
Done. @msagolj Grateful if you could update the JenkinsFile accordingly.